src/Symfony/Component/Form/DependencyInjection/FormPass.php

```diff
@@ -61,10 +65,33 @@ private function processFormTypes(ContainerBuilder $container): Reference
             }
         }
 
+        foreach ($container->findTaggedResourceIds('container.excluded.form.metadata.form_type') as $excludedServiceId => $tag) {
+            if (!isset($tag[0]['class_name'])) {
+                throw new InvalidArgumentException(\sprintf('The excluded service "%s" with tag "container.excluded.form.metadata.form_type" must have the tag\'s attribute "class_name" set.', $excludedServiceId));
+            }
+
+            $className = $tag[0]['class_name'];
```

Comment:
Here, you can get the class name from the service definition.

```suggestion
            $className = $container->getDefinition($excludedServiceId)->getClass();
```

Reply:
I'll check the behavior with this